src/tribler/core/components/metadata_store/restapi/tests/test_channels_endpoint.py

```diff
@@ -65,8 +65,7 @@ async def test_get_channels(rest_api, add_fake_torrents_channels, mock_dlmgr, me
     mock_dlmgr.download_exists = lambda *args: None
     json_dict = await do_request(rest_api, 'channels')
     assert len(json_dict['results']) == 10
-    # Default channel state should be METAINFO_LOOKUP
-    assert json_dict['results'][-1]['state'] == CHANNEL_STATE.METAINFO_LOOKUP.value
+    assert json_dict['results'][0]['state'] == CHANNEL_STATE.UPDATING.value
```

Comment:
It seems that our tests sometimes check multiple things at the same time. This test not only checks that it is possible to retrieve channels from the corresponding endpoint but also checks that a channel with local_version equal to zero has ` CHANNEL_STATE.METAINFO_LOOKUP` state by default.

**issue:** After the change, the fixture does not return any channel with the `METAINFO_LOOKUP` state anymore. Probably it still should.

**suggestion:** To fix it, the `subscribed` attribute formula should be `subscribed=bool(channel_i % 2 == 0)`, so `channel0` that have `local_version` zero is not subscribed. After that, the state of the last channel will be `CHANNEL_STATE.METAINFO_LOOKUP` again.

Reply:
Well, I don't change the internal logic of channels. 
This change only shows that with new (simplified) test data the state of the channel is `UPDATING` and `PREVIEW` instead of `METAINFO_LOOKUP` which shows that `METAINFO_LOOKUP` is not a default state.

The states of all channels in this test are following:

```python
00 = {str} 'Updating'
01 = {str} 'Updating'
02 = {str} 'Updating'
03 = {str} 'Updating'
04 = {str} 'Updating'
05 = {str} 'Updating'
06 = {str} 'Updating'
07 = {str} 'Updating'
08 = {str} 'Updating'
09 = {str} 'Preview'
```